bids/analysis/transformations/compute.py

```diff
@@ -33,12 +35,16 @@ def _transform(self, var, model='spm', derivative=False, dispersion=False,
 
         model = model.lower()
 
+        df = var.to_df(entities=False)
+
         if isinstance(var, SparseRunVariable):
-            sr = self.collection.sampling_rate
-            var = var.to_dense(sr)
+            sampling_rate = 10
+            resample_frames = np.arange(
+                0, var.get_duration(), 1/sampling_rate)
```

Comment:
From [the docs](https://docs.scipy.org/doc/numpy/reference/generated/numpy.arange.html#numpy.arange):

> When using a non-integer step, such as 0.1, the results will often not be consistent. It is better to use [numpy.linspace](https://docs.scipy.org/doc/numpy/reference/generated/numpy.linspace.html#numpy.linspace) for these cases.

I think the equivalent would be:

```Python
dur = var.get_duration()
resample_frames = np.linspace(0, dur, dur * sampling_rate,  endpoint=False)
```

Reply:
That explains some odd frames I got back. Thanks